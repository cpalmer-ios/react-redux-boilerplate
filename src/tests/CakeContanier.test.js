import { buyCake } from '../redux/cake/cakeActions';
import configureMockStore from 'redux-mock-store';

const mockStore = configureMockStore()
const store = mockStore({})
describe('action creators test to check the actions are matching the snapshots', () => {
    it('creates "BUY_CAKE" type when buyCake is dispatched', () => {
store.dispatch(buyCake());
expect(store.getActions()).toMatchSnapshot();
  });
});