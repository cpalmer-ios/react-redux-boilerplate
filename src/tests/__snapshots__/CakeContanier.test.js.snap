// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`action creators test to check the actions are matching the snapshots creates "BUY CAKE" type when buyCake is dispatched 1`] = `
Array [
  Object {
    "type": "BUY_CAKE",
  },
]
`;

exports[`action creators test to check the actions are matching the snapshots creates "BUY_CAKE" type when buyCake is dispatched 1`] = `
Array [
  Object {
    "type": "BUY_CAKE",
  },
]
`;

exports[`action creators test to check the actions are matching the snapshots creates BUY CAKE type when buyCake is dispatched 1`] = `
Array [
  Object {
    "type": "BUY_CAKE",
  },
]
`;
